gemfiles/rails_5.gemfile.lock

```diff
@@ -146,33 +149,33 @@ GEM
       byebug (~> 9.0)
       pry (~> 0.10)
     public_suffix (2.0.5)
-    rack (2.0.1)
+    rack (2.0.3)
     rack-protection (2.0.0)
       rack
     rack-ssl-enforcer (0.2.9)
     rack-test (0.6.3)
       rack (>= 1.0)
     rack-timeout (0.4.2)
-    rails (5.0.2)
-      actioncable (= 5.0.2)
-      actionmailer (= 5.0.2)
-      actionpack (= 5.0.2)
-      actionview (= 5.0.2)
-      activejob (= 5.0.2)
-      activemodel (= 5.0.2)
-      activerecord (= 5.0.2)
-      activesupport (= 5.0.2)
+    rails (5.1.1)
```

Comment:
Shouldn't this still be `5.0.x`? Because we already have [rails_5_1.gemfile.lock](https://github.com/deliveroo/roo_on_rails/blob/master/gemfiles/rails_5_1.gemfile.lock) for the 5.1 case.